storage/EtcCrypttab.h

```diff
@@ -199,6 +199,10 @@ namespace storage
          * Dump the current contents to the log.
          **/
         void log();
+
+    private:
+
+	static const int DEFAULT_PERMISSIONS;
```

Comment:
The value can be set right here, see https://en.cppreference.com/w/cpp/language/static.